mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -92,7 +93,15 @@ CLASS net/minecraft/class_1799 net/minecraft/item/ItemStack
 	METHOD method_7910 finishUsing (Lnet/minecraft/class_1937;Lnet/minecraft/class_1309;)Lnet/minecraft/class_1799;
 		ARG 1 world
 		ARG 2 user
-	METHOD method_7911 getOrCreateSubTag (Ljava/lang/String;)Lnet/minecraft/class_2487;
+	METHOD method_7911 getOrCreateNbt (Ljava/lang/String;)Lnet/minecraft/class_2487;
+		COMMENT Returns the {@linkplain NbtCompound} at the specified key in this item stack's custom NBT, or creates it if absent.
```

Comment:
```suggestion
		COMMENT {@return the compound NBT at the specified key in this item stack's NBT, or a new compound if absent} The modifications to the new compound are not stored into the stack while those to the existing compound are, so always call  {@link setNbt(String, NbtElement)} to save the changes you made.
```

A few points from this sample suggestion:
1. Don't use `{@linkplain NbtCompound}`, which will render like "[NbtCompound](http://example.com)" than "[`NbtCompound`](http://example.com)".
2. You don't have to add create extraneous links when the type referred to is part of the descriptor, such as param or return types. Javadoc links that for you
2. The create new behavior IMO is quite confusing, as I have fallen into the trap before (since modification on existing tag objects are saved, you would tend to think so for new tag objects)

Reply:
Uh, the modification part is quite weird.
Especially since the method code is:
```java
      if (this.nbt != null && this.nbt.contains(key, NbtElement.COMPOUND_TYPE)) {
         return this.nbt.getCompound(key);
      } else {
         var nbtCompound = new NbtCompound();
         this.setNbt(key, nbtCompound);
         return nbtCompound;
      }
```
So from that, it should be saved for new NBT compound objects.

Reply:
It does call `setNbt` now? Thanks mojang! then might note to users that this modifies the nbt and for pure reading purposes, use the nullable getter one.

Reply:
subTag is much clearer than just "nbt" which can now mean the serialized stack, the top level tag, or a subtag...